<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Nihar" failures="5" tests="12" name="Contact Functionality" time="145.944" errors="0" timestamp="2023-05-14T16:38:24 IST">
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_CharPhonNo" time="3.17">
    <failure type="java.lang.AssertionError" message="Phone number field should not contain characters expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Phone number field should not contain characters expected [true] but found [false]
at portfolio.test.ContactFunctionalityTest.validateWith_CharPhonNo(ContactFunctionalityTest.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validateWith_CharPhonNo -->
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_EmailEmpty" time="3.983"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_EmailWithoutSymbol" time="3.779"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_EmptyField" time="3.786"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_FullnameEmpty" time="3.737"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_InvalidEmail" time="4.793"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_InvalidEmail2" time="3.821"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_LengthyFullName" time="2.658">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at portfolio.test.ContactFunctionalityTest.validateWith_LengthyFullName(ContactFunctionalityTest.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateWith_LengthyFullName -->
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_LengthyPhoneNo" time="3.701">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at portfolio.test.ContactFunctionalityTest.validateWith_LengthyPhoneNo(ContactFunctionalityTest.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateWith_LengthyPhoneNo -->
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_MsgEmpty" time="24.047">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &amp;apos;By.xpath: //h2[text()=&amp;apos;Thank you for reaching out to me !&amp;apos;]&amp;apos; (tried for 10 second(s) with 500 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h2[text()='Thank you for reaching out to me !']' (tried for 10 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:87)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
at portfolio.abstractComponents.AbstractComponent.waitForElementToBeVisible(AbstractComponent.java:72)
at portfolio.pageObject.ThankUPage.getThankText(ThankUPage.java:22)
at portfolio.test.ContactFunctionalityTest.validateWith_MsgEmpty(ContactFunctionalityTest.java:99)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//h2[text()='Thank you for reaching out to me !']"}
  (Session info: chrome=113.0.5672.93)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.9.1', revision: 'eb2032df7f'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f50d5d7af81637bc9e57424cb64cae23, findElement {using=xpath, value=//h2[text()='Thank you for reaching out to me !']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.93, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\nihar\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:65079}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:65079/devtoo..., se:cdpVersion: 113.0.5672.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: f50d5d7af81637bc9e57424cb64cae23
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy17.isDisplayed(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:307)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:40)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:293)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:290)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
... 31 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateWith_MsgEmpty -->
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_PhoneNoEmpty" time="5.789"/>
  <testcase classname="portfolio.test.ContactFunctionalityTest" name="validateWith_ValidData" time="23.996">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &amp;apos;By.xpath: //h2[text()=&amp;apos;Thank you for reaching out to me !&amp;apos;]&amp;apos; (tried for 10 second(s) with 500 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h2[text()='Thank you for reaching out to me !']' (tried for 10 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:87)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
at portfolio.abstractComponents.AbstractComponent.waitForElementToBeVisible(AbstractComponent.java:72)
at portfolio.pageObject.ThankUPage.getThankText(ThankUPage.java:22)
at portfolio.test.ContactFunctionalityTest.validateWith_ValidData(ContactFunctionalityTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//h2[text()='Thank you for reaching out to me !']"}
  (Session info: chrome=113.0.5672.93)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.9.1', revision: 'eb2032df7f'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [91d656f091743eda5bca0ccb626313ae, findElement {using=xpath, value=//h2[text()='Thank you for reaching out to me !']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.93, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\nihar\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:65152}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:65152/devtoo..., se:cdpVersion: 113.0.5672.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 91d656f091743eda5bca0ccb626313ae
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy17.isDisplayed(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:307)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:40)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:293)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:290)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
... 31 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateWith_ValidData -->
</testsuite> <!-- Contact Functionality -->
